### PR TITLE
Fix some dependencies

### PR DESCRIPTION
**Pull request by imurchie**

### CODE REVIEW

File: lib/chromedriver.js [hunk 1]
```diff
@@ -32,7 +33,7 @@ class Chromedriver extends events.EventEmitter {
   async initChromedriverPath () {
     if (this.executableVerified) return;
     let binPath = this.chromedriver || (await getChromedriverBinaryPath());
-    if (!(await exists(binPath))) {
+    if (!(await fs.exists(binPath))) {
```
Comment (jlipps) [inline lib/chromedriver.js]: didn't we decide that we would do `(!await fs.exists(binPath))` since precedence allows us to?

File: lib/utils.js [hunk 2]
```diff
@@ -1,9 +1,8 @@
-import Q from 'q';
-import fs from 'fs';
+import { fs } from 'appium-support';
 
 async function exists (p) {
```
Comment (jlipps) [inline lib/utils.js]: shouldn't we be using appium-support's exists method instead of this one now?

File: lib/chromedriver.js (hunk 1 above)
Comment (imurchie) [inline lib/chromedriver.js]: Did we? I didn't know. I'll go through for it (though this is not intended to be a full clean up, just attempting to get the package to install correctly).

File: lib/utils.js (hunk 2 above)
Comment (imurchie) [inline lib/utils.js]: We are. I just forgot to delete this file.

**Pull request merged**